frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -1,38 +1,20 @@
 import React, { useEffect } from 'react'
 import styled from 'styled-components'
-import TopNavBar from '../../../components/Mobile/TopNavBar'
-import bookingsIcon from '../../../assets/bookingsIcon.svg'
-import messageIcon from '../../../assets/messageIcon.svg'
-import adminIcon from '../../../assets/adminIcon.svg'
 import { useParams } from 'react-router-dom'
 import { useDispatch, useSelector } from 'react-redux'
 import { useHistory } from 'react-router-dom'
 import 'react-date-range/dist/styles.css' // main css file
 import 'react-date-range/dist/theme/default.css' // theme css file
-import Button from '../../../components/Mobile/Button'
+import Button from '../../../components/Button'
 import BottomNavBar from '../../../components/Mobile/BottomNavBar'
-import { Alert } from 'antd'
 import 'antd/dist/antd.css'
 import { PATHS } from '../../Routes'
 import { RootState } from '../../../store/types'
-import {
-  createNewBookingFromFacility,
-  fetchFacilityNameFromID,
-  getAllBookingsForFacility,
-  SetIsLoading,
-  setViewDates,
-  setSelectedFacility,
-} from '../../../store/facilityBooking/action'
-import { months } from '../../../common/dates'
-import LoadingSpin from '../../../components/LoadingSpin'
-import { DOMAIN_URL, ENDPOINTS } from '../../../store/endpoints'
+import { fetchFacilityNameFromID, SetIsLoading, setSelectedFacility } from '../../../store/facilityBooking/action'
```

Comment:
help update SetIsLoading to setIsLoading

Reply:
For the entire app?